backend/api/Models/Concepts/AcquisitionFile/AcquisitionFileModel.cs

```diff
@@ -27,11 +27,6 @@ public class AcquisitionFileModel : FileModel
         /// </summary>
         public DateTime? DeliveryDate { get; set; }
 
-        /// <summary>
-        /// get/set - The acquisition file status type.
-        /// </summary>
-        public TypeModel<string> AcquisitionFileStatusTypeCode { get; set; }
-
```

Comment:
No longer needed here. This model inherits from `FileModel` that provides a shared property for file status